### PR TITLE
[FIX] stock_picking_batch_extended: float_is_zero required parameters

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa TT18083

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Comment by i-vyshnevska** [comment]
can you update other calls float_is_zero in this addon

File: stock_picking_batch_extended/report/batch_report.py [hunk 1]
```diff
@@ -33,20 +33,24 @@ def new_level_0(self, operation):
             'l1_items': {},
         }
 
+    @api.model
+    def _get_operation_qty(self, operation):
+        return float_is_zero(
+            operation.product_qty,
+            precision_rounding=operation.product_uom_id.rounding
+        ) and operation.qty_done or operation.product_qty
```
Comment (chienandalu) [inline stock_picking_batch_extended/report/batch_report.py:41]: Should not be qty_done tested with float_is_zero as well?

Comment (chienandalu) [reply stock_picking_batch_extended/report/batch_report.py:41]: Commented offline :)

**Review by chienandalu — APPROVED** [review]

**Review by sergio-teruel — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@i-vyshnevska is it ok for you?

**Review by i-vyshnevska — APPROVED** [review]

**Pull request merged**